platformcomponents/android/slider.json

```diff
@@ -0,0 +1,33 @@
+{
+  "slider": {
```

Comment:
iOS slider is at https://github.com/momentum-design/tokens/blob/master/platformcomponents/ios/slider.json - although there's no technical reason, it might be worth trying to match token names.

Reply:
If possible I would like to keep these names, since they were chosen to match the Material Design Slider API which is being used for the android toolkit slider component

Reply:
but I will talk to Emir about it to see how close we can get